ansible/roles/operator-pipeline/templates/openshift/tasks/acquire-lock.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: acquire-lease
+spec:
+  params:
+  - name: lease-name
+    type: string
+  - name: timeout
+    type: string
+    default: 90m
+  stepTemplate:
+    env:
+      - name: TOTAL_DURATION_IN_SECONDS
+        value: ${0}
+  steps:
+    - name: create-lease
+      image: image-registry.openshift-image-registry.svc:5000/openshift/cli:latest
```

Comment:
Is this pullspec always available in the cluster? Can you check a Red Hat Catalog and find the same image in the official Red Hat registry?

Reply:
I will check for the image from the Red Hat Catalog and update it here.